charts/tidb-drainer/templates/_helpers.tpl

```diff
@@ -10,12 +14,19 @@ config-file: |-
     {{- if .Values.config }}
 {{ .Values.config | indent 2 }}
     {{- end -}}
+    {{- if .Values.enableTLSCluster }}
```

Comment:
Suggest keeping the same logic with pump, no need to get `security` from values.yaml since users cannot change them anyway.

Reply:
done